test/playwright/test_ux.py

```diff
@@ -10,7 +10,7 @@
 import random
 
 
-DEFAULT_TIMEOUT = 30_000  # time in ms
+DEFAULT_TIMEOUT = 60_000  # time in ms
```

Comment:
I increased this value because I saw "Timeout 30000ms exceeded" in the GitHub Action logs